lms/static/sass/base/_base.scss

```diff
@@ -130,7 +130,6 @@ a:focus {
   margin: 0 auto 0;
   padding: ($baseline*2) 0;
   max-width: grid-width(12);
-  min-width: 760px;
```

Comment:
@talbs 

1. To make the section in which the course cards are contained responsive. Problem occurred for screen sizes lesser than 760px.
2. I will resolve this by adding more specific class for this section, something like `courses-container`. That way, I will be sure that I didn't break anything else in the platform while adjusting the section that I want to make responsive.